Setup.bat

```diff
@@ -111,34 +114,46 @@ call :MarkStartOfBlock "Create folders"
 call :MarkEndOfBlock "Create folders"
 
 call :MarkStartOfBlock "Retrieve dependencies"
-    spatial package retrieve tools           schema_compiler-x86_64-win32               %PINNED_CORE_SDK_VERSION%   %DOMAIN_ENVIRONMENT_VAR%   "%CORE_SDK_DIR%\tools\schema_compiler-x86_64-win32.zip"
-    spatial package retrieve schema          standard_library                           %PINNED_CORE_SDK_VERSION%   %DOMAIN_ENVIRONMENT_VAR%   "%CORE_SDK_DIR%\schema\standard_library.zip"
-    spatial package retrieve worker_sdk      c_headers                                  %PINNED_CORE_SDK_VERSION%   %DOMAIN_ENVIRONMENT_VAR%   "%CORE_SDK_DIR%\worker_sdk\c_headers.zip"
-    spatial package retrieve worker_sdk      c-dynamic-x86_64-vc141_md-win32            %PINNED_CORE_SDK_VERSION%   %DOMAIN_ENVIRONMENT_VAR%   "%CORE_SDK_DIR%\worker_sdk\c-dynamic-x86_64-vc141_md-win32.zip"
-    spatial package retrieve worker_sdk      c-dynamic-x86_64-clang1000-linux           %PINNED_CORE_SDK_VERSION%   %DOMAIN_ENVIRONMENT_VAR%   "%CORE_SDK_DIR%\worker_sdk\c-dynamic-x86_64-clang1000-linux.zip"
+    call :ExecuteAndCheck spatial package retrieve tools         schema_compiler-x86_64-win32               %PINNED_CORE_SDK_VERSION%    %DOMAIN_ENVIRONMENT_VAR%    "%CORE_SDK_DIR%\tools\schema_compiler-x86_64-win32.zip"
+    if ERRORLEVEL 1 pause && exit /b %ERRORLEVEL%

```

Comment:
is this not repeating what you do in :ExecuteAndCheck ?

Reply:
`:ExecuteAndCheck` will catch, report and propagate an error(code) but cannot quit execution of the batch (afaik `exit` can only exit the scope it is called in), which is why similar checks are needed
_Just another super helpful and fun feature of batch_ /s

Reply:
'afaik exit can only exit the scope it is called in' <- I don't think that is true, can't find that behaviour in any syntax reference and a quick check on my end confirms that exit exits from inside an if statement. Are you a 100% sure this works on your end?

Reply:
Also I don't think the if statement constitutes a scope in batch, I think that is controlled by SETLOCAL / ENDLOCAL only

Reply:
Ah yes /b switch returns control to the batchfile so calling `exit` without should exit the process

Reply:
Many thanks, will give that a go :)